provers/mock/prover.go

```diff
@@ -27,6 +28,9 @@ func NewProver(chain core.Chain, config ProverConfig) *Prover {
 }
 
 func (pr *Prover) Init(homePath string, timeout time.Duration, codec codec.ProtoCodecMarshaler, debug bool) error {
+	if logger := log.GetLogger(); logger != nil {
```

Comment:
I think we should not check `logger != nil`.
If it is `nil`, the program panics, but it's a desirable result.

Reply:
fixed